session/session.go

```diff
@@ -138,12 +138,13 @@ func New(args ...interface{}) *Session {
 	envFallback("SOFTLAYER_TIMEOUT", &values[keys["timeout"]])
 
 	// Read ~/.softlayer for configuration
-	u, err := user.Current()
+	dir, err := homedir.Dir()
+	fmt.Println(err)
```

Comment:
- [ ] Did you intend to leave that here?  
  At this point, we don't know whether `err` is nil or not.